src/main/java/frc/robot/subsystems/gripper/GripperLoggedInputs.java

```diff
@@ -0,0 +1,24 @@
+package frc.robot.subsystems.gripper;
+
+import org.littletonrobotics.junction.LogTable;
+import org.littletonrobotics.junction.inputs.LoggableInputs;
+
+public class GripperLoggedInputs implements LoggableInputs {
+    public boolean solenoidState;
+
+    /**
+     * Puts parameters into the table.
+     */
+    @Override
+    public void toLog(LogTable table) {
+        table.put("solenoid", solenoidState);
```

Comment:
Please rename from "solenoid", probably there will be more solenoids and it will make it harder to distinguish between them

Reply:
I'm pretty sure the logs are organized by subsystem, so it's fine (although this solenoid is really the only one on the robot).
However, this name should be more descriptive. It doesn't log the solenoid itself, but whether the gripper is closed or not.

Reply:
okay

Reply:
done.